test/lib/util/fast_json_encoder_test.exs

```diff
@@ -0,0 +1,61 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.FastJasonEncoderTest do
+  use Croma.TestCase
+  use ExUnitProperties
+
+  def validate(value) do
+    validate(value, value)
+  end
+  def validate(source, expected) do
```

Comment:
Functions with an arity of 1 and 2 will have a blank line.

Reply:
Fixed. b0c7c48381997e5ac476ebf9b332e49eab348fd4